qutip/solver/solver_base.py

```diff
@@ -284,10 +284,17 @@ def sys_dims(self):
         """
         Dimensions of the space that the system use:
 
-        ``qutip.basis(sovler.dims)`` will create a state with proper dimensions
+        ``qutip.basis(solver.dims)`` will create a state with proper dimensions
```

Comment:
Let's fix the other error too:
```suggestion
        ``qutip.basis(solver.sys_dims)`` will create a state with proper dimensions
```

And maybe add a note it's not sufficient for exotic dimensions?